mesh/profiles/views.py

```diff
@@ -1,22 +1,49 @@
 from django.core.exceptions import ObjectDoesNotExist
-from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
+from django.core import serializers
+from mesh.accounts.models import Account
 
 from mesh.profiles.models import Profile
 
+from django.views import View
 
-def bio_view(request):
-    if request.method == "POST":
-        data = request.POST
-        if 'accountID' not in data:
-            return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
-        if 'biography' not in data:
-            return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
+class BiographyView(View):
+    """
+    View for getting an biography by accountId or updating an biography by accountID.
+    """
+    def get(self, request, account_id, *args, **kwargs):
+        """
+        Handle GET requests.
+
+        Retrieves a biography of the listed profile in JSON format.
+
+        Returns a JSON response containing biography of specified profile through id.
+        """
+        try:     
+            if(account_id is None):
+                return JsonResponse({'error': 'Invalid input. Missing account ID'}, status=401)
+            
+            profile = Profile.objects.get(accountID=account_id)
+            profileBiography = profile.biography
+            return JsonResponse({'message': profileBiography}, safe=False, status=200)
+        except Account.DoesNotExist:
+            return JsonResponse({'error': 'Invalid request. Account does not exist'}, status=404)
+        except Profile.DoesNotExist:
+            return JsonResponse({'error': 'Invalid request. Profile does not exist'}, status=402)
```

Comment:
This status code should likely also be `status = 404` since 402 is `402 Payment Required`.